python-exercises/assignment/parking_lot.py

```diff
@@ -0,0 +1,239 @@
+#!/usr/bin/env python3
+
+#This is  parking lot assignment code file
+
+import sys
+
+from prettytable import PrettyTable
+
+class Car:
+    """
+    Class of Car 
+    """
+    def __init__(self, registration_number, color):
+        # car registration number
+        self.registration_number = registration_number
+        # car color
+        self.color = color
+
+class Slot:
+    """
+    Class of Slot
+    """
+    def __init__(self, id):
+        # slot id
+        self.id = id
+        # car in current slot
+        self.car = None
+
+    def park_car(self, car: Car):
+        """
+        This function is to park a car in a slot
+        """
+        status = ''
+        if self.car is None:
+            self.car = car
+            status = 'OK'
+        else:
+            status = 'This slot is not empty!'
```

Comment:
The `string` help in case the user/developer look into the log, is there a better way to allow who use this function to check if they have successfully park the car?

Imagine I'm doing this

```
car = Car("1234", "Black")
some_slot = Slot(x)
success = some_slot.park_car(car)
if not success:
    another_slot = Slot(y)
(vice versa)
```